neuro_san/service/mcp/handlers/mcp_root_handler.py

```diff
@@ -172,8 +172,12 @@ async def post(self):
                         self.agent_policy)
                 call_params: Dict[str, Any] = data.get("params", {})
                 tool_name: str = call_params.get("name")
-                prompt: str = call_params.get("arguments", {}).get("input", "")
-                result_dict: Dict[str, Any] = await tools_processor.call_tool(request_id, metadata, tool_name, prompt)
+                call_args: Dict[str, Any] = call_params.get("arguments", {})
+                prompt: str = call_args.get("input", "")
+                chat_context: str = call_args.get("chat_context", None)
+                sly_data: Dict[str, Any] = call_args.get("sly_data", None)
+                result_dict: Dict[str, Any] =\
+                    await tools_processor.call_tool(request_id, metadata, tool_name, prompt, chat_context, sly_data)
                 self.set_status(HTTPStatus.OK)
```

Comment:
In addition to "input", which is user prompt, extract also "chat context" and "sly data" parameters from MCP tool call request.